src/app/public/modules/timepicker/timepicker.directive.ts

```diff
@@ -154,8 +169,11 @@ export class SkyTimepickerInputDirective implements
 
   public writeValue(value: any) {
     this.modelValue = this.formatter(value);
+    this._validatorChange();
+    this._onChange(this.modelValue);
```

Comment:
Could you remind me why we are calling `_onChange` here? I thought that was what triggered `writeValue` to be called. Is this not causing a loop or too many `valueChanges` to be fired?

Reply:
Agreed. The `_onChange` should be called when the value changes internally, and needs to notify the consuming `FormGroup`.